website/docs/r/policy_assignment.html.markdown

```diff
@@ -79,6 +79,10 @@ The following arguments are supported:
 
 * `policy_definition_id` - (Required) The ID of the Policy Definition to be applied at the specified Scope.
 
+* `identity_type` - (Optional) The managed identity type associated with the policy assignment. Either `None` or `SystemAssigned`. Change forces a new resource. Will be mandatory if you assign a policy with `deployIfNotExists` effect.
```

Comment:
(as below) since this'll now be in a block we should be able to make this:

```
* `identity` - (Optional) An `identity` block as defined below. Changing this forces a new resource to be created.

...

An `identity` block supports the following:

* `type` - (Required) The Managed Service Identity Type of this Policy Assignment. At this time the only supported value is `SystemAssigned`.
```

Reply:
👍